wcivf/apps/parties/management/commands/import_parties.py

```diff
@@ -6,7 +6,13 @@
 
 class Command(BaseCommand):
     def handle(self, **options):
-        next_page = settings.YNR_BASE + "/api/next/parties/?page_size=200"
+        next_page = (
+            settings.YNR_BASE
+            + "/api/next/parties/?page_size=200?"
+            + "&auth_token="
+            + settings.YNR_API_KEY
+        )
```

Comment:
f string?

Reply:
It renders correctly without the f string